core/src/main/scala/io/finch/endpoint/body.scala

```diff
@@ -21,7 +21,11 @@ private abstract class FullBody[A] extends Endpoint[A] {
       val output = Rerunnable.fromFuture {
         val contentLength = input.request.contentLengthOrNull
         if (contentLength == null || contentLength == "0") missing
-        else present(input.request.contentType, input.request.content, input.request.charsetOrUtf8)
+        else present(
+          Option(input.request.mediaTypeOrNull),
```

Comment:
Given that we always wrap this in an option, maybe we want to push that down into `mediaTypeOrNull` and have that function return an `Option[String]`?

Reply:
Yeah, I was thinking about this. Maybe even eliminate the need for an option wrapper and just treat empty strings as none? This would shove some allocations from the request path.